yeonjae/programmers-0/삼각형의 완성조건.js

```diff
@@ -0,0 +1,15 @@
+function solution(sides) {
+  const maxLength = Math.max(...sides);
+  const minLength = Math.min(...sides);
+  const resultArr = [];
+
+  for (let i = maxLength - minLength + 1; i <= maxLength; i++) {
+    resultArr.push(i);
+  }
+
+  for (let j = maxLength + minLength - 1; j > maxLength; j--) {
+    resultArr.push(j);
+  }
+
+  return resultArr.length;
+}
```

Comment:
```suggestion
function solution(sides) {
  const maxLength = Math.max(...sides);
  const minLength = Math.min(...sides);
  const result = 0;

  for (let i = maxLength - minLength + 1; i <= maxLength; i++) {
    result++;
  }

  for (let i = maxLength + minLength - 1; i > maxLength; i--) {
    result++;
  }

  return result;
}
```
- 풀이 잘봤습니다! resultArr의 배열을 활용해도 풀 수 있지만 원시값 숫자를 이용해서도 풀 수 있다는점 참고해주시길 바랍니다. 🙇‍♂️
- 그리고 for문에서 사용하는 `i`, `j`, `k` 등은 중첩 반복문에서 구분하기 위함입니다. 위와 같이 중첩 for문이 아닌 연달아 for문을 작성할 때는 i로 작성하시는 것을 추천합니다.

```js
// 중첩 for문
  for (let i = 0; i < arr.length; i++) {
    for (let j = 0; j < arr.length; j++) {
      for (let k = 0; k < arr.length; k++) {
        // 코드
      }
    }
  }
```
```js
// 그냥 for문 2개 쓰기
for (let i = 0; i < arr.length; i++) {}
for (let i = 0; i < arr.length; i++) {}
```

Reply:
넵 확인했습니다! 습관처럼 i, j사용한거같아요 ㅎㅎ,, 피드백 감사해요!